### PR TITLE
[cus][WIP] CuS Binding initial contribution

### LINKED ISSUE #5025: [cus] Add CuS Binding
Dear Community,

I am working on a binding for C&S - a company which improves care through innovative services. Stay tuned.

Best regards,
Jochen

### PR DESCRIPTION
**Pull request by Jochen1980**
The binding connects to services of C&S Computer and Software AG, Augsburg, Germany ( https://www.managingcare.de ).
Signed-off-by: Jochen Bauer <jochen.bauer@gmail.com> (github: Jochen1980)

Resolves #5025

### CODE REVIEW

**Comment by Jochen1980** [comment]
Should I try to fix that, or does the repo maintainer wants to do that? Thanks for the feedback?

File: addons/binding/org.openhab.binding.cus/src/main/java/org/openhab/binding/cus/internal/CuSHandler.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+ *
+ * @author Jochen_Bauer - Initial contribution
+ */
+@NonNullByDefault
+public class CuSHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(CuSHandler.class);
+
+    @Nullable
+    private CuSConfiguration config;
```
Comment (davidgraeff) [inline addons/binding/org.openhab.binding.cus/src/main/java/org/openhab/binding/cus/internal/CuSHandler.java]: Please use the pattern "private @Nullable CuSConfiguration config;"

**Comment by davidgraeff** [comment]
At the moment, you will get once in a while a merge conflict with "addons/binding/pom.xml" and also CODEOWNERS (which you forgot to add your WIP binding to).

It is also recommended that you switch over to the new buildsystem at some point (but implementing your binding has higher priority), because we are not accepting bindings anymore that works the old way :)

**Comment by J-N-K** [comment]
@Jochen1980, do you intend to finish this one?

**Comment by Jochen1980** [comment]
Of course, we will finish that one. In the upcoming months we will be able to spend more time on openHAB-related work packages. The reason for the delay was / is that we are not used to the new commit procedure after the change from 2.4 to 2.5. Give us  a few weeks then, it should be done.

**Comment by Hilbrand** [comment]
Closing because remote repository is gone. It also only contains the skeleton generated code.

**Pull request closed without merging**